client/src/helpers/APICalls/getAllUnreadNotifications.ts

```diff
@@ -0,0 +1,13 @@
+import { NotificationApiData } from '../../interface/NotificationApiData';
+
+const getUnread = async (): Promise<NotificationApiData> => {
+  const userId = '619c1eb37a1e963a5b179c4b';
```

Comment:
Please remove the hardcoded user id here, once we add the `protect` middleware to the routes - we can determine the id of the logged in user :)